consensus/reactor.go

```diff
@@ -43,26 +44,29 @@ type Reactor struct {
 
 	conS *State
 
-	mtx      cmtsync.RWMutex
-	waitSync bool
+	waitSync atomic.Bool
 	eventBus *types.EventBus
-	rs       *cstypes.RoundState
+
+	rs    *cstypes.RoundState
+	rsMtx cmtsync.Mutex
```

Comment:
```suggestion
	rsMtx cmtsync.Mutex
	rs    *cstypes.RoundState
```

Our convention is usually to list the mutex variable just above all the other variables it guards. I don't see any issues with renaming it.